apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ClientPool.cs

```diff
@@ -45,10 +45,12 @@ internal static class ClientPool
         }
 
         //ReSharper disable once UnusedMember.Global
-        //Returns true if all client reference counts are zero.
+        //Returns the total of all reference counts.
         //For test purposes only.
-        internal static int GetTotalClientRefCount(StringBuilder poolContents)
+        // poolContents will contain the current contents of the pool and may not be null.
```

Comment:
Nit: I'd use "...will be filled with..." like you do below. This is a bit ambiguous as to whether it is expected to contain the contents or whether it will contain it after the call returns.